apps/docs/content/guides/database/extensions/pgvector.mdx

```diff
@@ -6,6 +6,10 @@ description: 'pgvector: a PostgreSQL extension for storing embeddings and perfor
 
 [pgvector](https://github.com/pgvector/pgvector/) is a PostgreSQL extension for vector similarity search. It can also be used for storing [embeddings](https://supabase.com/blog/openai-embeddings-postgres-vector).
 
+<Admonition type="note">
+  The name of pgvector's Postgres extension is [vector](https://github.com/pgvector/pgvector/blob/258eaf58fdaff1843617ff59ea855e0768243fe9/README.md?plain=1#L64).
```

Comment:
@gregnr @charislam just making sure that users are aware the actual name is `vector` since there was confusion in support tickets, etc.